coalib/output/Logging.py

```diff
@@ -4,25 +4,40 @@
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def configure_logging():
+
+def configure_logging(new_log_level=logging.INFO, incremental=False):
     """
     Configures the logging with hard coded dictionary.
     """
     import sys
 
     logging.config.dictConfig({
         'version': 1,
+        'incremental': incremental,
```

Comment:
When `configure_logging` is called for the first time, the increment should be `False`, when using it to update the levels we set it to `True`, thus this change.